spec.js

```diff
@@ -0,0 +1,38 @@
+describe('Successful login:', function(){
```

Comment:
Please update to have a single describe with two it blocks (positive and negative)

Reply:
@nbabii,
Updated with a single describe (with 2 it blocks).
Added package.json file.
Please review changes.